Realm/Realm.SourceGenerator/Diagnostics.cs

```diff
@@ -271,6 +272,15 @@ public static Diagnostic TypeNotSupported(string className, string propertyName,
                 location);
         }
 
+        public static Diagnostic NestedClass(string className, Location location)
+        {
+            return CreateDiagnosticError(
+                Id.NestedClass,
+                "Nested classes are not supported yet.",
+                $"{className} is a nested class, which is not yet supported.",
```

Comment:
Can we emit this as `{parentClass}.{className}`? Not critical, I just feel like it'll make it more obvious that it's nested and where it's nested in.

Reply:
Fixed